lib/watson.js

```diff
@@ -0,0 +1,190 @@
+var request = require('request');
+var jsonfile = require('jsonfile');
+
+var DiscoveryV1 = require('watson-developer-cloud/discovery/v1');
+config = jsonfile.readFileSync(__dirname + '/config.json' );
```

Comment:
you should be able to go one folder back here and find the config file in the top level directory, where they are usually located by doing:

config = jsonfile.readFileSync(__dirname + '/../config.json');